devel/google-cloud-sdk/Portfile

```diff
@@ -20,22 +20,22 @@ supported_archs     i386 x86_64
 
 if { ${configure.build_arch} eq "i386" } {
     distname        ${name}-${version}-darwin-x86
-    checksums       rmd160  c8139a7e87702ab9933979d99093454288f2fb3e \
-                    sha256  2895762c646e3a8decc95a92cf9b09a426318d59efb19a8100cc3bd5444592fe \
-                    size    86065585
+    checksums       rmd160  fef40eeab167fc88c6efc6b30959f41049268ca9 \
+                    sha256  d8f11372d7b69e19c6e320cdbd4f3b86fad44bbd72406a7bb5eca85389b9f4d3 \
+                    size    86395433
 } elseif { ${configure.build_arch} eq "x86_64" } {
     distname        ${name}-${version}-darwin-x86_64
-    checksums       rmd160  40b7e494f3560c465faea32c95f9b3196c529092 \
-                    sha256  3bca7104f35cd0eacab475b2828cf979dc4a488ac5213c4cceb9d7b104dc38d6 \
-                    size    108128649
+    checksums       rmd160  3931fba11b5807d47257b25b73432bbb93652897 \
+                    sha256  17db4d373eb2607fc1e6d6c49881d0370b9e02415f52a35654a48e78233b3f7e \
+                    size    108978062
 }
 
 homepage            https://cloud.google.com/sdk/
 master_sites        https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/
 
 worksrcdir          ${name}
 
-python.default_version 38
+python.default_version 39
```

Comment:
According to https://cloud.google.com/sdk/docs/install Python 3.5 to 3.8 are currently supported, which is why I kept this at 3.8.

Reply:
I see. The original idea behind the bump was so that it can be built on BigSur/aarch64 (python38+universal won't build whilst python39+universal does). I did not bother hunting down why python38 won't build as the sdk (gcloud gsutil bq) "works on my machine" using python39, even though it is not officially supported yet: https://issuetracker.google.com/issues/170125513

Reply:
The comments on https://issuetracker.google.com/issues/170125513 indicate that there are a couple of actual issues with Google Cloud SDK on Python 3.9, so I don't think it's a good idea to make this the default yet.

Reply:
Agreed, never mind then